calm_adapter/terraform/worker.tf

```diff
@@ -0,0 +1,19 @@
+module "worker" {
+  source = "../../infrastructure/modules/worker"
+
+  name = local.namespace
+
+  image              = local.calm_adapter_image
+  env_vars           = local.env_vars
+  secret_env_vars    = local.secret_env_vars
+  min_capacity       = 1
+  max_capacity       = 2
+  desired_task_count = 1
```

Comment:
Why doesn't the CALM adapter scale down to 0 when it's not running?

Reply:
This is what I did previously so that's how I set it up.

In practice it looks like we have `desired_task_count = 1` for pretty much all our catalogue applications:

https://github.com/wellcomecollection/catalogue/blob/master/pipeline/terraform/modules/worker/variables.tf#L29-L32
https://github.com/wellcomecollection/catalogue/blob/master/sierra_adapter/terraform/modules/scaling_worker/variables.tf#L28-L30

(we don't explicitly set any of these when using the modules).